Modules/Core/SpatialObjects/include/itkSpatialObjectProperty.h

```diff
@@ -55,6 +55,12 @@ class ITKSpatialObjects_EXPORT SpatialObjectProperty
     m_Color = color;
   }
 
+  ColorType
+  Color()
+  {
+    return m_Color;
+  }
+
   ColorType &
   GetColor()
```

Comment:
@aylward Is a

Reply:
See comments used to close this commit.

Reply:
@aylward I did see your comments, I'm just trying to understand the issue. Apparently returning a `Color` reference seems to work fine, while some tweaks may be necessary to return an `std::string` reference. I did not know about that before.